bootstrap.yml

```
@@ -72,29 +87,34 @@
-      url="https://dev.azure.com/${organization}/_apis/distributedtask/pools/${poolId}/agents?includeCapabilities=true&api-version=7.1"
-
-      response=$(curl -s -w "%{http_code}" "${headers[@]}" -X GET "$url")
-      http_code=${response: -3}
-      content=${response::-3}
-
-      if [ $http_code -eq 200 ]; then
-          # Extract all userCapabilities names for online and enabled agents as a unique list
-          capabilityNames=$(echo "$content" | jq -r '[.value[] | select(.status == "online" and .enabled == true) | .userCapabilities | keys] | unique | flatten | join("\n- ")')
+      if [ $skipHardwareTest -eq true ]; then
+        # Set the capabilityNames to none
+        capabilityNames="none";        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)